traffic_control/mixins/models.py

```diff
@@ -69,8 +69,28 @@ class Meta:
         abstract = True
 
 
-class OwnedDeviceModel(models.Model):
+class ResponsibleEntityModel(models.Model):
```

Comment:
If there's no mentions of "Responsible Entity" stuff left in `traffic_control`, should this belong to `city_furniture` instead?